pyproject.toml

```diff
@@ -70,9 +70,7 @@ array-api-extra = { path = ".", editable = true }
 
 [tool.pixi.feature.lint.dependencies]
 pre-commit = "*"
-pylint = "*"
-basedmypy = "*"
```

Comment:
replaced basedmypy with mypy. I think it is an extremely bad idea to diverge from PEPs. Namely, IDE plugins will not support this.

Reply:
This is untrue. All mypy IDE plugins and other CI tools also work with `basedmypy`. 

In fact, mypy diverges more from the typing specification that `basedmypy` does. It also has [significantly more bugs](https://github.com/erictraut/mypy_issues).